Add Screenshot guidelines to the Marketing Handbook (#269)
chatwoot/docs: docs/handbook/marketing/guidelines-taking-product-screenshots-or-screencasts.md

    - Is understanding every little step or screen in a defined process essential to the user?
    - Which one do you think would be more attention-grabbing?
    - Which format complements the context of the solution better?

## General Guidelines

Here are some basic things to keep in mind: 

- Focus on the problem to be solved and deliver the right message through your images. Focus on the main part of the screenshot.
- Do not show the entire window if it is not adding any value to the screenshot.
- You can choose to zoom in on the part of the window that is core to the message you’re trying to convey or very lightly blur the background to help the viewer know which part of the image to focus on.
- Use a good contrast if you’re sketching or annotating on the screenshot. Ideally, one should understand the intended message of a screenshot at the first glance.
- Direct the eye of the reader. Use arrows if necessary.
- Do not hesitate to include more screenshots if it helps in getting the point across more clearly. Otherwise, record a screencast or GIF.
    
    ### **Let’s see an example**
    

>     I want to illustrate “how to add a personal message signature.” I’ll give you four options below:

Author: ```suggestion
    Let us illustrate “how to add a personal message signature.” Look at the four options below:
```

    
    **I.**
    
    ![Untitled](https://s3-us-west-2.amazonaws.com/secure.notion-static.com/8c7d22cd-f0e2-47a4-a1d3-57a77d270f59/Untitled.png)
    
    **II.**
    
    ![Untitled](https://s3-us-west-2.amazonaws.com/secure.notion-static.com/98046b6e-5535-408b-88cf-aa0c9acf8398/Untitled.png)
    
    III.
    
    ![Untitled](https://s3-us-west-2.amazonaws.com/secure.notion-static.com/0a45b07f-994b-4be0-b11c-97116a23e293/Untitled.png)
    
    IV.
    
    ![Untitled](https://s3-us-west-2.amazonaws.com/secure.notion-static.com/540bf6f8-cefa-4216-af46-cd8f251d3694/Untitled.png)
    
    ***So, which one did you select?***
    

>     I’m pretty sure you would’ve selected the fourth option because it checks all the boxes. Let’s see what those are:

Author: ```suggestion
    We’re pretty sure you would’ve selected the fourth option because it checks all the boxes. Let’s see what those are:
```

    
    - [ ]  At a single glance, you can tell where the image wants you to focus.
    - [ ]  It gives you an idea about the entire screen you’d be viewing when finding this setting, while not compromising with the core message. As opposed to this, SS #3 wasn’t that helpful.
    - [ ]  All the unnecessary info is blurred, but only so much that you can see/read what the options above and below the “personal message signature” are but they wouldn’t eat up space in the screenshot and confuse you. This’ll help you locate this option more easily. However, notice that the side panel and “Account settings” are not blurred. It’s because these are important pieces of info for this particular screenshot.
    - [ ]  A red rectangle is added around the area I want you to focus on.
    

>     ***Alternatively***, if I had to create a gif, I would make something like this: 

Author: ```suggestion
    ***Alternatively***, a gif should look something like this: 
```

## Approach to be followed

Having the right approach will take away all the guesswork from creating a screenshot. This will only take a couple of minutes. So, begin by asking yourself the following questions: 

1. **Which channel am I publishing to?** 
Are you creating the screenshot/screencast for the help center, a landing page, a social media post, or a deck? Having this clarity will make the rest of the process easier. 

2. **Who is the audience?** 
If you’re creating this for a Twitter post, the target audience is the following we have there. These are generally startup decision-makers, competitors, folks in the industry, well-wishers, community folks and even our users. If you’re creating the screenshot for the help center, you can be sure of the fact that these are our existing users who are looking for a specific answer. So, take a minute to determine who you’re creating the screenshot for so that you can tailor-make it for them and provide the best value possible.

> The brand color for Chatwoot is **#1f93ff**

Author: ```suggestion
```
